__tests__/app.test.js

```diff
@@ -488,11 +501,20 @@ describe("DELETE /api/user_id/reviews/:review_id", () => {
   });
   test("should return a 400 when given an invalid user ID", () => {
     return request(app)
-      .delete("/api/fake-user-ID/reviews/63ce75449ae462be0adae13a")
-      .expect(404)
+      .delete("/api/fake-user-ID/reviews/totally-a-real-user-id")
```

Comment:
This test has both a fake user id and review id. Think the review id should be real in this case.